components/script/dom/globalscope.rs

```diff
@@ -3121,7 +3121,7 @@ impl GlobalScope {
     /// <https://www.w3.org/TR/gamepad/#dfn-gamepadconnected>
     pub fn handle_gamepad_connect(
         &self,
-        index: usize,
+        _index: usize,
```

Comment:
Let's add a note about what we discussed re indices, and why we are ignoring it for now.